src/test/java/sqlline/PromptTest.java

```diff
@@ -30,6 +31,15 @@
  * Test cases for prompt and right prompt.
  */
 public class PromptTest {
+  private static final String DEV_NULL = "/dev/null";
+  private SqlLine sqlLine;
+
+  @BeforeEach
+  private void init() {
+    sqlLine = new SqlLine();
+    sqlLine.getOpts().setPropertiesFile(DEV_NULL);
+  }
+
   @Test
   public void testPromptWithoutConnection() {
     SqlLine sqlLine = new SqlLine();
```

Comment:
Since `sqlLine` becomes as private field, this line is  removable.

Reply:
Thanks for the catch, removed all these lines from this file.
There are some more tests with such lines but there is no sense to change as those tests do not look into properties